### ISSUE TITLE
Inquiry about LoRA SFT Settings for Qwen2.5-VL in Seg-Zero

### ISSUE THREAD

**Issue by Terence233** [dvlab-research/Seg-Zero#86]
Hi, thanks for the great work!

I have a quick question regarding the SFT baseline experiment using LoRA on Qwen2.5-VL-3B.

Could you kindly share more details on the LoRA configuration used for SFT? Specifically:

- LoRA rank (r) and alpha values?
- Which modules/layers of Qwen2.5-VL were injected with LoRA adapters? (e.g., attention/query/key/value/MLP?)
- Was bias tuning enabled?
- What was the optimizer and learning rate for the LoRA parameters?
- How many training steps/epochs were used?

It would be of great help and looking forward to your reply!

**Comment by LiuRicky** [comment]
Thanks for your interest. 

We use fully fine-tune SFT baseline, instead of LoRA fine-tuning. We train 600 steps (SFT) in total and evalute the best ckpt performance.

**Comment by Dingyi211** [comment]
Hello, I would like to ask if your training settings in SFT and RL training are consistent? For example, the learning rate.

**Comment by LiuRicky** [comment]
The training steps and data are the same.

For learing rate, we use both lr=1e-5 (default in SFT) and lr=1e-6 (same as RL), and choose the best performed ckpts.

**Comment by Dingyi211** [comment]
Hello, could I ask again, are the batch sizes set to the same in both experiments? Or is the total batch size for sft set to 32?

**Comment by LiuRicky** [comment]
Sorry I did not remember, but I just use the batch size that full the GPU memory.
